cpu/esp32/Makefile.include

```diff
@@ -164,7 +164,11 @@ LINKFLAGS += -L$(RIOTCPU)/$(CPU)/ld/$(CPU_FAM)/
 ifeq (,$(filter esp_ble,$(USEMODULE)))
```

Comment:
If only `esp_ble_esp32` requires the special linker script, why not
```suggestion
ifeq (,$(filter esp_ble_esp32,$(USEMODULE)))
```

Reply:
`esp_ble_esp32` is only a feature, but we could use
```
ifeq (,$(filter esp32_sdk_lib_bt_esp32,$(USEPKG)))
```
instead.

Reply:
I thought features get turned into modules at the end of dependency resolution, but that will also work.